contracts/cw721-base/src/state.rs

```diff
@@ -65,11 +65,7 @@ impl<'a> IndexList<TokenInfo> for TokenIndexes<'a> {
 
 pub fn tokens<'a>() -> IndexedMap<'a, &'a str, TokenInfo, TokenIndexes<'a>> {
     let indexes = TokenIndexes {
-        owner: MultiIndex::new(
-            |d, k| (Vec::from(d.owner.as_ref()), k),
-            "tokens",
-            "tokens__owner",
-        ),
+        owner: MultiIndex::new(|d, k| (d.owner.clone(), k), "tokens", "tokens__owner"),
```

Comment:
Do you need clone here? Isn't it `|d: TokenInfo, k: Addr|`? Or is it `|d: &TokenInfo, k: Addr|`?

Inlining the proper types would definitely make this more readable if we use it as an learning example

Reply:
It's the second one. That's why `clone` is needed.

Will do the type inlining.